jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/actions/OpenStreamInEditor.kt

```diff
@@ -27,9 +29,12 @@ object OpenStreamInEditor {
     }
 }
 
-fun List<OutputLogEvent>.buildStringFromLogs() = buildString {
+// This is named differently because buildStringFromLogs with two different lists has the same type on JVM, yay type erasure
```

Comment:
What if we inline them? Does it still suffer from the issue?

Reply:
Still suffers this issue. We could annotate them with different `@JvmName` names but I think that's probably not a better solution

Reply:
Ya, oh well